toolbox

```diff
@@ -143,7 +143,7 @@ spinner_start()
 
     (
         while [ -f "$directory/spinner-start" ]; do
-            echo "$spinner_animation" | sed "s/ /\n/g" 2>&3 | while read frame; do
+            echo "$spinner_animation" | sed "s/ /\n/g" 2>&3 | while read -r frame; do
```

Comment:
++

I merged this patch to silence SC2162 via #89 after some testing. Thanks Tomas!